Product/complete-example-of-a-rule-processor-configuration.md

```diff
@@ -4,86 +4,77 @@
 
 ```
 {
-  "id": "006@1.0.0",
-  "cfg": "1.0.0",
-  "desc": "Outgoing transfer similarity - amounts",
-  "config": {
-    "parameters": {
-      "maxQueryLimit": 3,
-      "tolerance": 0.1
-    },
-    "exitConditions": \[
-      {
-        "subRuleRef": ".x00",
-        "outcome": false,
-        "reason": "Incoming transaction is unsuccessful"
+    "id": "006@1.0.0",
+    "cfg": "1.0.0",
+    "desc": "Outgoing transfer similarity - amounts",
+    "config": {
+      "parameters": {
+        "maxQueryLimit": 3,
+        "tolerance": 0.1
       },
-      {
-        "subRuleRef": ".x01",
-        "outcome": false,
-        "reason": "Insufficient transaction history"
-      }
-    \],
-    "bands": \[
-      {
-        "subRuleRef": ".01",
-        "upperLimit": 2,
-        "outcome": false,
-        "reason": "No similar amounts detected in the most recent transactions from the debtor"
-      },
-      {
-        "subRuleRef": ".02",
-        "lowerLimit": 2,
-        "upperLimit": 3,
-        "outcome": true,
-        "reason": "Two similar amounts detected in the most recent transactions from the debtor"
-      },
-      {
-        "subRuleRef": ".03",
-        "lowerLimit": 3,
-        "outcome": true,
-        "reason": "Three or more similar amounts detected in the most recent transactions from the debtor"
-      }
-    \]
+      "exitConditions": \[
```

Comment:
Please update - not sure where these artefacts came from, but I suspect something in the migration process is trying to escape the square brackets (for no apparent reason...)